adi_update_tools.sh

```diff
@@ -241,24 +241,21 @@ rfsom_box ()
 }
 
 # Allow selective builds by default build the latest release branches
-if [ "$1" = "dev" ]
+if [ "$1" = "dev" -o "$1" = "master" ]
 then
   BUILDS=$BUILDS_DEV
-elif [ "$1" = "2019_R1" ]
+elif [ "$1" = "2019_R1" -o "$1" = "2019_r1" ]
 then
   BUILDS=$BUILDS_2019_R1
-elif [ "$1" = "2019_R2" ]
+elif [ "$1" = "2019_R2" -o "$1" = "2019_r2" ]
 then
   BUILDS=$BUILDS_2019_R2
-elif [ "$1" = "2021_R1" ]
+elif [ "$1" = "2021_R1" -o "$1" = "2021_r1" ]
 then
   BUILDS=$BUILDS_2021_R1
-elif [ "$1" = "2021_R2" ]
+elif [ "$1" = "2021_R2" -o "$1" = "2021_r2" ]
 then
   BUILDS=$BUILDS_2021_R2
-elif [ -n "$1" ]
-then
-  BUILDS=$1
```

Comment:
I don't think that making the parameter case insensitivity is a bad thing, however, removing the option of manually providing the branch names it's unrelated to this commit and anyway, it's a useful option that should remain available.

Reply:
v2: this case was added back.